frameworks/PHP/symfony/symfony-raw.dockerfile

```diff
@@ -7,30 +7,31 @@ RUN LC_ALL=C.UTF-8 add-apt-repository ppa:ondrej/php > /dev/null && \
     apt-get update -yqq > /dev/null && apt-get upgrade -yqq > /dev/null
 
 RUN apt-get install -yqq nginx git unzip curl \
-    php8.2-cli php8.2-fpm php8.2-mysql  \
+    php8.2-cli php8.2-fpm php8.2-pgsql  \
     php8.2-mbstring php8.2-xml php8.2-curl > /dev/null
 
-RUN curl -sSL https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer
+COPY --from=composer:latest /usr/bin/composer /usr/local/bin/composer
 
 COPY deploy/conf/* /etc/php/8.2/fpm/
-RUN if [ $(nproc) = 2 ]; then sed -i "s|pm.max_children = 1024|pm.max_children = 512|g" /etc/php/8.2/fpm/php-fpm.conf ; fi;
 
+ADD . /symfony
 WORKDIR /symfony
-ADD ./composer.json /symfony/
+
+RUN if [ $(nproc) = 2 ]; then sed -i "s|pm.max_children = 1024|pm.max_children = 512|g" /etc/php/8.2/fpm/php-fpm.conf ; fi;
+
 RUN mkdir -m 777 -p /symfony/var/cache/{dev,prod} /symfony/var/log
-RUN COMPOSER_ALLOW_SUPERUSER=1 composer install --no-dev --no-scripts
-ADD . /symfony
-RUN COMPOSER_ALLOW_SUPERUSER=1 composer dump-autoload --no-dev --classmap-authoritative
-RUN COMPOSER_ALLOW_SUPERUSER=1 composer dump-env prod
 
-# removes hardcoded option `ATTR_STATEMENT_CLASS` conflicting with `ATTR_PERSISTENT`. Hack not needed when upgrading to Doctrine 3
-# see https://github.com/doctrine/dbal/issues/2315
-#RUN sed -i '/PDO::ATTR_STATEMENT_CLASS/d' ./vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php
+ENV COMPOSER_ALLOW_SUPERUSER=1
+RUN composer install --optimize-autoloader --classmap-authoritative --no-dev --quiet --no-scripts
+RUN cp deploy/postgresql/.env . && composer dump-env prod && bin/console cache:clear
 
-RUN php bin/console cache:clear 
 RUN echo "opcache.preload=/symfony/var/cache/prod/App_KernelProdContainer.preload.php" >> /etc/php/8.2/fpm/php.ini
 
 EXPOSE 8080
 
-CMD service php8.2-fpm start && \
```

Comment:
Why don't use service ??

Reply:
service should be already started at this point!

Reply:
Well we can use restart.

Reply:
I really don't get it, here we are building the image, so the service will start when container will run with the correct configuration, or am I wrong? I never saw the need to start the service manually!

Reply:
It's only simplicity:
`service php8.2-fpm start`
vs
`/usr/sbin/php-fpm8.2 --fpm-config /etc/php/8.2/fpm/php-fpm.conf`